keycloak/client.go

```diff
@@ -358,6 +381,54 @@ func (c Client) getGroup(ctx context.Context, token *gocloak.JWT, toSearch Group
 	return find(g), nil
 }
 
+func (c Client) getChildGroups(ctx context.Context, token *gocloak.JWT, groupID string) ([]gocloak.Group, error) {
+	var result []*gocloak.Group
+	childGroupsUrl := strings.Join([]string{c.Host, "admin", "realms", c.Realm, "groups", groupID, "children"}, "/")
+	resp, err := c.Client.GetRequestWithBearerAuth(ctx, token.AccessToken).
+		SetResult(&result).
+		Get(childGroupsUrl)
+
+	if err != nil {
+		return nil, &gocloak.APIError{
+			Code:    0,
+			Message: errors.Wrap(err, "could not retrieve child groups").Error(),
```

Comment:
Nit: The `error.Wrap` does not make much sense here since only the string is used anyways.